### PR TITLE
Add logs to a test case

### PR DESCRIPTION
**Pull request by dushansilva**

### CODE REVIEW

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/restapi/ContentSearchTestCase.java [hunk 1]
```diff
@@ -470,6 +502,7 @@ private String getAccessToken(String clientName, String username, String passwor
         org.wso2.carbon.automation.test.utils.http.client.HttpResponse tokenGenerateResponse = HttpRequestUtil
                 .doPost(tokenApiUrl, messageBody, accessKeyMap);
         JSONObject tokenGenJsonObject = new JSONObject(tokenGenerateResponse);
+        log.info("Token Response: " + tokenGenJsonObject.toString());
```
Comment (bhathiya) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/restapi/ContentSearchTestCase.java]: Please call `String getResponseBody(HttpResponse response)` here and print only the token response body.

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/restapi/ContentSearchTestCase.java [hunk 2]
```diff
@@ -100,15 +101,16 @@
...
                 Assert.assertTrue(true);
                 break;
             } else {
                 if (i == retries) {
-                    Assert.fail("Basic content search in publisher failed. Received response : " + publisherResponse);
+                    Assert.fail("Basic content search in publisher failed. Received response : " + responseString);
                 } else {
-                    log.warn("Basic content search in publisher failed. Received response : " + publisherResponse + " Retrying...");
-                    Thread.sleep(3000);
+                    log.warn("Basic content search in publisher failed. Received response : " + responseString + " Retrying...");
```
Comment (AmaliMatharaarachchi) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/restapi/ContentSearchTestCase.java]: Shall we check whether the length of this line exceeding our formatting rules?

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/restapi/ContentSearchTestCase.java [hunk 3]
```diff
@@ -123,15 +125,16 @@
...
                 Assert.assertTrue(true);
                 break;
             } else {
                 if (i == retries) {
-                    Assert.fail("Basic content search in store failed. Received response : " + storeResponse);
+                    Assert.fail("Basic content search in store failed. Received response : " + responseString);
                 } else {
-                    log.warn("Basic content search in store failed. Received response : " + storeResponse + " Retrying...");
-                    Thread.sleep(3000);
+                    log.warn("Basic content search in store failed. Received response : " + responseString + " Retrying...");
```
Comment (AmaliMatharaarachchi) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/restapi/ContentSearchTestCase.java]: Length of the line

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/restapi/ContentSearchTestCase.java (hunk 2 above)
Comment (dushansilva) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/restapi/ContentSearchTestCase.java]: fixing in https://github.com/wso2/product-apim/pull/5009

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/restapi/ContentSearchTestCase.java (hunk 3 above)
Comment (dushansilva) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/restapi/ContentSearchTestCase.java]: fixing in https://github.com/wso2/product-apim/pull/5009

**Pull request merged**